index_v1.html

```diff
@@ -1,42 +1,101 @@
+<?php
+$currentPage = 'login_1';
+include 'nav-login.php';
+
+$error = '';

```

Comment:
**Error Handling:** The code checks if there's an error parameter in the URL query string (e.g., login.php?error=Invalid%20credentials). If this parameter exists, it captures the error message, using htmlspecialchars() to ensure that any potentially harmful characters are escaped, preventing XSS attacks. This makes the error message safe to display on the page.